### DISCUSSION TITLE
Color for brackets?

### DISCUSSION THREAD

**Discussion by sorrowlol** [tiagovla/tokyodark.nvim#15] [Ideas]
Hi, I really like this colorscheme, I've been using it a lot! 

Is there anyway to add colors for brackets? I tried to play around with the settings of other plugs for bracket colors but they all ruin the style :(

Thank you!

**Comment by tiagovla** [comment]
Hi. Do you want to change the color of all brackets or do you want like a rainbow thing? If you have a plugin that it's not working, I can check what's happening.

**Reply by sorrowlol** [reply]
Hi :) I was looking for a rainbow thing. The plugin I was trying to use was https://github.com/luochen1990/rainbow but the colours really don't fit the style and I couldn't find how to change these.

**Comment by tiagovla** [comment]
I would try this treesitter one https://github.com/p00f/nvim-ts-rainbow instead.

**Reply by sorrowlol** [reply]
I went through this plugin earlier, but for some reason Treesitter was confusing to install. I will give it another try, thanks!
